README.md

```diff
@@ -1,86 +1,44 @@
+### The overall instruction is described [here](https://github.com/Juniper/contrail-tripleo-heat-templates/blob/master/README.md). Follow it with the changes below:
 
-**Define a puppet-contrail node**
+* If you wnat to use [CentOS based images](https://cloud.centos.org/centos/7/images/) you will have to build own overcloud images for Compute nodes with Trove service excluded. This is because Trove depends on python2-xml2dict package that conflicts with the custom Contrail package xmltodict-0.7.0-0contrail.el7.noarch.rpm.
 
-```puppet
-class { 'contrail': }
+* At the step 'Undercloud configuration / get contrail' download Contrail v4.0 install packages and docker containers
+(there is an assumption that you have acces to Juniper build server 10.84.5.120)
 ```
-
-Implementation
---------------
-
-### puppet-contrail
-
-Puppet-contrail is a combination of Puppet manifest and ruby code to delivery configuration and extra functionality through types and providers.
-
-Limitations
-------------
-
-*
-
-Beaker-Rspec
-------------
-
-This module has beaker-rspec tests
-
-To run the tests on the default vagrant node:
-
-```shell
-bundle install
-bundle exec rake acceptance
+wget http://10.84.5.120/github-build/R4.0/LATEST/redhat70/newton/contrail-install-packages-4.0.0.0-20~newton.el7.noarch.rpm
+sudo rpm -ivh contrail-install-packages-4.0.0.0-20~newton.el7.noarch.rpm
+sudo tar -zxvf /opt/contrail/contrail_packages/contrail_rpms.tgz -C /var/www/html/contrail
+wget http://10.84.5.120/github-build/R4.0/LATEST/redhat70/newton/contrail-docker-images_4.0.0.0-20.tgz
+sudo tar -zxvf contrail-docker-images_4.0.0.0-20.tgz -C /var/www/html/contrail/
 ```
 
-For more information on writing and running beaker-rspec tests visit the documentation:
-
-* https://github.com/puppetlabs/beaker/wiki/How-to-Write-a-Beaker-Test-for-a-Module
-
-Development
------------
-
-Developer documentation for the entire puppet-openstack project.
-
-* https://wiki.openstack.org/wiki/Puppet-openstack#Developer_documentation
-
-Contributors
-------------
+* At the step 'Configure overcloud / get puppet modules' use this puppet-contrail module.
+```
+mkdir -p ~/usr/share/openstack-puppet/modules
+git clone https://github.com/Juniper/contrail-tripleo-puppet -b stable/newton ~/usr/share/openstack-puppet/modules/tripleo
+git clone https://github.com/alexey-mr/puppet-contrail -b stable/newton ~/usr/share/openstack-puppet/modules/contrail
+tar czvf puppet-modules.tgz usr/
+```
 
-* https://github.com/sbadia/puppet-contrail/graphs/contributors
+*  If you use CentOS based images add openstack-utils packet into Contrail rpm repo. It is needed since CentOS images don't have this packet installed but Contrail requires it.
+```
+sudo wget -P /var/www/html/contrail  http://mirror.comnet.uz/centos/7/cloud/x86_64/openstack-newton/common/openstack-utils-2017.1-1.el7.noarch.rpm
+sudo createrepo --update -v /var/www/html/contrail
```

Comment:
createrepo does not seem to exist in my RHEL7.3 and must be installed explicitly with:
yum install createrepo

probably these 2 steps need their own explanation vs being in the codeblock for opemstack-utils.